sources/PreferenceInfo.h

```diff
@@ -12,6 +12,7 @@ typedef NS_ENUM(NSInteger, PreferenceInfoType) {
     kPreferenceInfoTypeCheckbox,
     kPreferenceInfoTypeInvertedCheckbox,  // true=checked, false=unchecked. Handy when inverting a checkbox's text, but the user defaults key can't be changed.
     kPreferenceInfoTypeIntegerTextField,
+    kPreferenceInfoTypeFloatTextField,
```

Comment:
Let's use Double, not Float. The extra precision doesn't come with a significant price.